Amazon.Ion.ObjectMapper/IonDictionarySerializer.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Amazon.IonDotnet;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Amazon.Ion.ObjectMapper
+{
+    public class IonDictionarySerializer : IonSerializer<IDictionary>
```

Comment:
Assuming this code is complete and PR only missing tests, please add docs.

Reply:
As in docstrings? It would be odd if that is in the scope of this PR arbitrarily as we do not yet have docstrings for anything else.

It would be appropriate as a part of the stylecop/styling PR we're planning for later.

Reply:
I disagree - I've been asking for docs on public facing methods in recent reviews. The cleanup PR should address shortcomings, but we should not compound the problem by omitting them now.

Reply:
Done.